src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component< {}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handlerKeyEvent);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handlerKeyEvent);
+  }
+
+  handlerKeyEvent = (event: KeyboardEvent) => {
```

Comment:
Function name should start from a verb:
```suggestion
  handleKeyPress = (event: KeyboardEvent) => {
```